### ISSUE TITLE
Start multiple servers in jest-dev-server

### ISSUE THREAD

**Issue by neoziro** [smooth-code/jest-puppeteer#134]
As suggested in #94, the idea is to be able to launch several servers using `jest-dev-server`. To do it we will be able to specify an array instead of an object:

```js
// jest-puppeteer.config.js
module.exports = {
  server: [
    {
      command: 'node server.js',
      port: 4444,
    },
    {
      command: 'node server2.js',
      port: 4445,
    },
  ]
}
```

**Comment by xiaoyuhen** [comment]
hi,neoziro.
Has this feature been developed?
If you haven’t developed yet, maybe I can contribute this feature on this weekend.

**Comment by neoziro** [comment]
Hello @xiaoyuhen, this feature has not been deployed. You can contribute on it if you want 🤗

**Comment by neoziro** [comment]
Fixed in #163
